src/bliss/util/args.h

```diff
@@ -0,0 +1,65 @@
+#ifndef ARGS_H
+#define ARGS_H
```

Comment:
It's a little annoying, but we might want to name the def `BLISS_ARGS_H`. If another library uses `ARGS_H` already in their code and we import, it will end up with a conflict. We can thank legacy build systems for this